### ISSUE TITLE
Adding ABIEncoderV2 pragma to source code returns exit 1 and does not build contracts

### ISSUE THREAD

**Issue by snario** [EthWorks/Waffle#30]
I tried compiling on the latest version (2 beta 0) the following:

```solidity
pragma solidity 0.5;

contract Test {

  enum ActionTypes { INCREMENT, DECREMENT}

  struct Action {
    ActionTypes actionType;
    uint256 byHowMuch;
  }

  struct AppState {
    address player1;
    address player2;
    uint256 count;
    uint256 turnNum;
  }
}
```

and it worked however when adding

```solidity
pragma experimental ABIEncoderV2;
``` 

No build files are created.

This warning is emitted, which is expected:

```
contracts/Test.sol:3:1: Warning: Experimental features are turned on. Do not use experimental features on live deployments.
pragma experimental ABIEncoderV2;
^-------------------------------^
```

**Comment by snario** [comment]
I added a fix in #31

**Comment by marekkirejczyk** [comment]
Great job. Thank you!
